index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styel.css" />
+  </head>
+  <body>
+    <div class="nav-container">
+      <div class="wrapper">
+        <nav>
+          <h2 class="logo">Logo</h2>
+          <img  class="image" src="Enabled.svg" alt="" />
+        </nav>
+      </div>
+    </div>
+    <div class="header-container">
+      <div class="wrapper">
+        <header>
+          <div class="typography">
+            <h1>
+              I’m Mahdia <br />
+              Glad to see you!
+            </h1>
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <a href="">LET’S CONNECT</a>
+          </div>
+          <div class="img">
+            <img src="twitter.svg" class="twitter" alt="" />
+            <img src="linked.svg" class="linkedIn" alt="" />
+            <img src="medium.svg" class="meduim" alt="" />
+            <img src="angellist.svg" class="angellist" alt="" />
```

Comment:
- [ ] Kindly ensure that your UI looks exactly like the original design from Figma. you can add the background like on the Figma template by taking a screenshot of that background. Also, make sure that you use the same `font-style` and `font-size` like as on the [original design from Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

|Figma Design| Your Design |
|---------------| --------------|
|![image](https://user-images.githubusercontent.com/71874425/185486432-99c0f1fc-4de0-471a-b00f-3d40535d8670.png)|![image](https://user-images.githubusercontent.com/71874425/185486546-6f61f4d1-376f-443d-8066-337ffc81b099.png)|